ipatests/test_integration/test_installation.py

```diff
@@ -2226,3 +2227,42 @@ def test_forcesync_does_not_overwrite_ignore_time_skew(
         conn = self.replicas[0].ldap_connect()
         ldap_entry = conn.get_entry(DN("cn=config"))
         assert ldap_entry.single_value['nsslapd-ignore-time-skew'] == "on"
+
+
+class TestInstallKeySizes(IntegrationTest):
```

Comment:
Can you also add a job for this new test class in the nightly definitions in ipatests/prci_definitions/ (nightly_latest_pki.yaml, nightly_latest_selinux.yaml, nightly_latest_testing_selinux.yaml, nightly_latest_testing.yaml, nightly_latest.yaml, nightly_previous.yaml, nightly_rawhide.yaml) and test it in a temp commit?